src/shared/utils/git-utils.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { BitbucketIcon } from '@patternfly/react-icons/dist/js/icons/bitbucket-icon';
+import { GitAltIcon } from '@patternfly/react-icons/dist/js/icons/git-alt-icon';
+import { GithubIcon } from '@patternfly/react-icons/dist/js/icons/github-icon';
+import { GitlabIcon } from '@patternfly/react-icons/dist/js/icons/gitlab-icon';
+
+export enum GitProvider {
+  GITHUB = 'github',
+  BITBUCKET = 'bitbucket',
+  GITLAB = 'gitlab',
+  UNSURE = 'other',
+  INVALID = '',
+}
+
+export const gitUrlRegex =
```

Comment:
We have gitUrlRegex already in `src/components/ImportForm/utils/validation-utils.ts`